test/samples/pipeline/test_ray_pipeline.yaml

```diff
@@ -0,0 +1,46 @@
+version: 'test-version'
+
+components:
+  - name: Reader
+    type: FARMReader
+    params:
+      no_ans_boost: -10
+      model_name_or_path: deepset/roberta-base-squad2
+      num_processes: 0
+  - name: ESRetriever
+    type: ElasticsearchRetriever
+    params:
+      document_store: DocumentStore
+  - name: DocumentStore
+    type: ElasticsearchDocumentStore
+    params:
+      index: haystack_test
+      label_index: haystack_test_label
+  - name: PDFConverter
+    type: PDFToTextConverter
+    params:
+      remove_numeric_tables: false
+  - name: Preprocessor
+    type: PreProcessor
+    params:
+      clean_whitespace: true
+  - name: IndexTimeDocumentClassifier
+    type: TransformersDocumentClassifier
+    params:
+      batch_size: 16
+      use_gpu: false
+  - name: QueryTimeDocumentClassifier
+    type: TransformersDocumentClassifier
+    params:
+      use_gpu: false
+
+
+pipelines:
+  - name: ray_query_pipeline
+    type: RayPipeline
```

Comment:
Validation will fail if a pipeline has a type as I understand it?

Reply:
Yes, but Ray pipelines are not validated (yet), so as long as this YAML is read via `RayPipeline.load_from_yaml()` it will work as before. It's to be fixed though, thanks for spotting it :slightly_smiling_face: